packages/core-data/src/entities.js

```diff
@@ -228,6 +232,7 @@ export const rootEntitiesConfig = [
 		kind: 'root',
 		baseURL: '/wp/v2/plugins',
 		baseURLParams: { context: 'edit' },
+		plural: 'plugins',
```

Comment:
Can we somehow guarantee that if we add a new entity in the future it will also have a `plural` defined?

Reply:
It's not a required property. Maybe converting the file to TS and adding proper types could help us here.

Reply:
Since we keep the `plural` check, we should be good 👍